PetaPoco/Core/MultiPocoFactory.cs

```diff
@@ -90,8 +90,13 @@ private static Delegate FindSplitPoint(Type typeThis, Type typeNext, string conn
             for (; pos < r.FieldCount; pos++)
             {
                 // Split if field name has already been used, or if the field doesn't exist in current poco but does in the next
-                var fieldName = r.GetName(pos);
-                if (usedColumns.ContainsKey(fieldName) || (!pdThis.Columns.ContainsKey(fieldName) && pdNext.Columns.ContainsKey(fieldName)))
+                var fieldName = r.GetName(pos);
+                var pdThisWithTableName = pdThis.Columns.FirstOrDefault(x => pdThis.TableInfo.TableName + "." + x.Key == fieldName);

```

Comment:
Can you explain what this is for? It's been a while since I've looked at MultiPocoFactory